src/rpc.ts

```diff
@@ -132,6 +132,12 @@ router.post('/api/scores', async (req, res) => {
   )
     return rpcError(res, 500, 'something wrong with the strategies', null);
 
+  try {
+    addresses.forEach(getAddress);
+  } catch (e: any) {
+    return rpcError(res, 400, 'invalid address', null);
+  }
+
```

Comment:
Not sure if it will break something 😄 on the playground we still pass a lot of lowercased addresses from examples.json of strategies. will try bit deep tomorrow

Reply:
we can change examples.json later